schemas/context/profile-other-work-details.example.1.json

```diff
@@ -0,0 +1,40 @@
+{
+  "xdm:otherAddress": {
+    "@id": "https://data.adobe.io/entities/address/234",
+    "xdm:primary": false,
+    "xdm:street1": "100 Park Ave",
+    "xdm:city": "San Jose",
+    "xdm:stateProvince": "US-CA",
+    "xdm:postalCode": "95110",
+    "xdm:country": "United States",
+    "xdm:countryCode": "US",
+    "schema:latitude": 37.3382,
+    "schema:longitude": 121.8863,
+    "xdm:status": "active",
+    "xdm:lastVerifiedDate": "2018-01-02"
+  },
+  "xdm:otherPhone": {
+    "xdm:primary": true,
+    "xdm:number": "1-408-888-7777",
+    "xdm:status": "active"
+  },
+  "xdm:assistantDetails": {
+    "xdm:name": {
+      "xdm:firstName": "John",
+      "xdm:middleName": "S",
+      "xdm:lastName": "Doe",
+      "xdm:fullName": "John S. Doe"
+    },
+    "xdm:phone": {
+      "xdm:primary": true,
+      "xdm:number": "1-408-888-7777",
+      "xdm:status": "active"
+    }
+  },
+  "xdm:reportsToID": "RPT10000",
```

Comment:
Should be in profile-work-details

Reply:
This was in the profile-work-details before. I moved these to the new mixin after we discussed.

Reply:
We discussed just for "Other" categories, rest were okay to be related work